test/javascript/tests/bulk_docs.js

```diff
@@ -110,6 +110,29 @@ couchTests.bulk_docs = function(debug) {
   T(result.error == "bad_request");
   T(result.reason == "POST body must include `docs` parameter.");
 
+
+  // verify that sending a request with invalid `docs` causes error
+  var req = CouchDB.request("POST", "/" + db_name + "/_bulk_docs", {
+    body: JSON.stringify({"docs": "foo"})
+  });
+
+  T(req.status == 400);
+  result = JSON.parse(req.responseText);
+  T(result.error == "bad_request");
+  T(result.reason == "`docs` parameter must be an array.");
```

Comment:
This test looks unrelated to code change. Shouldn't it come with separate commit?

Reply:
@kxepal I've split the commit in two.

Reply:
Thanks!